diff --git a/next.config.ts b/next.config.ts
index 7da1bc6..6069cd9 100644
--- a/next.config.ts
+++ b/next.config.ts
@@ -20,7 +20,6 @@ const nextConfig: NextConfig = {
     return [
       { source: "/apps", destination: "/shop", permanent: true },
       { source: "/apps/:slug", destination: "/shop/:slug", permanent: true },
-      { source: "/dashboard", destination: "/downloads", permanent: true },
     ];
   },
 };
